internal/handlers/telegram/handler_test.go

```diff
@@ -0,0 +1,28 @@
+package telegram
+
+import (
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestDocument(t *testing.T) {
+	correct := "test shared a file (https://teleirc.com/file.txt) on Telegram with caption: 'Random Caption'."
```

Comment:
Good, but due to my comment above, we don't actually want to show the file link. 
I would recommend just doing:
```
correct := "test shared a file on Telegram with caption: 'Random Caption'"
```
when there is no mimetype available, as we don't want to show the bot api token.